ui/react-components/containers/AppointmentContainer.jsx

```diff
@@ -2,45 +2,32 @@ import React, {Component} from "react";
 import AddAppointment from "../components/AddAppointment/AddAppointment.jsx";
 import {IntlProvider} from "react-intl";
 import {getLocale} from "../utils/LocalStorageUtil";
-import {getTranslations} from "../api/translationsApi";
-import {getAppConfigs} from "../api/configApi";
 import translations from '../../app/i18n/appointments';
-import {appName} from '../constants';
 import PropTypes from "prop-types";
 import {AppContext} from "../components/AppContext/AppContext";
+import {getAppConfig, getMessages} from "../components/AppContext/AppService";
 import EditAppointment from "../components/EditAppointment/EditAppointment.jsx";
 import moment from "moment";
 
 // TODO : need to add connection to redux
 
 class AppointmentContainer extends Component {
 
-    constructor (props) {
+    constructor(props) {
         super(props);
         this.state = {
             locale: getLocale(),
             messages: translations[props.locale],
             appConfig: null
         };
         (async () => {
-            await this.getMessages();
-            await this.getAppConfigs();
+            this.setState({messages: await getMessages(this.state.locale)});
```

Comment:
can we use react hooks for state?

Reply:
I dont think react hooks can be used in class component.